pkgs/desktops/cinnamon/mint-y-icons/default.nix

```diff
@@ -18,6 +19,7 @@ stdenv.mkDerivation rec {
 
   propagatedUserEnvPkgs = [
```

Comment:
I think `propagatedBuildInputs` should be enough but I am not completely sure.

Reply:
This derivation was already using `propagatedUserEnvPkgs` and I have just added missing dependencies. I will change it to `propagatedBuildInputs`, handling the dependencies uniformly.

Reply:
Is `propagatedUserEnvPkgs` documented somewhere? I am not finding in on the [nixpkgs manual](https://nixos.org/nixpkgs/manual/).

Reply:
It is not documented because it is frowned upon.

But reading https://github.com/NixOS/nixpkgs/issues/34657#issuecomment-363465755, `propagatedBuildInputs` might not work.